amqp/README.md

```diff
@@ -1,51 +1,41 @@
-AMQP 0.9.1 API Reference
-========================
+# API Reference
 
-If you don't know what AMQP is you can read [this](https://blogs.vmware.com/vfabric/2013/02/choosing-your-messaging-protocol-amqp-mqtt-or-stomp.html)
-and [this](https://www.rabbitmq.com/tutorials/amqp-concepts.html).
+TTN makes use of an AMQP Topic exchange (`amq.topic`). See [this](https://www.rabbitmq.com/tutorials/amqp-concepts.html) for details.
 
-TTN use the Topic AMQP exchange. The default exchange name is `amq.topic`
+AMQP is not yet available in the public network. The main reasons for this are: 
+_security_, _scalability_ and the AMQP protocol not being _standardised_ yet.
```

Comment:
It is, but not the broker that we're using. AMQP 1.0 is a decent, standardised protocol. ActiveMQ is a nice broker for it, but we don't have a Go client for it